tldw_Server_API/app/api/v1/endpoints/mcp_unified_endpoint.py

```
@@ -33,6 +33,13 @@
 from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
 from tldw_Server_API.app.core.AuthNZ.settings import is_single_user_mode, get_settings
 from tldw_Server_API.app.core.MCP_unified.security.request_guards import enforce_http_security
+from tldw_Server_API.app.api.v1.API_Deps.auth_deps import (
+    require_permissions,
+    get_auth_principal,
+)
+from tldw_Server_API.app.core.AuthNZ.permissions import SYSTEM_LOGS
+from tldw_Server_API.app.core.AuthNZ.principal_model import AuthPrincipal
+from tldw_Server_API.app.core.MCP_unified.auth.jwt_manager import TokenData
```

Ruff (0.14.7)

Redefinition of unused `TokenData` from line 30

Remove definition: `TokenData`

(F811)